libs/order/src/models/order-invoice.ts

```diff
@@ -0,0 +1,14 @@
+import { DaffOrderAddress } from './order-address';
+import { DaffOrderPayment } from './order-payment';
+import { DaffOrderShippingMethod } from './order-shipping-method';
+import { DaffOrderShipmentItem } from './order-shipment-item';
+import { DaffOrderTotal } from './order-total';
+
+export interface DaffOrderInvoice {
+  items: DaffOrderShipmentItem[];
```

Comment:
`DaffOrderInvoiceItem`

Reply:
Is that just an alias or do we need a separate interface entirely?

Reply:
This would just be an alias for now.

Reply:
changed